repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/KeystoneV2FilterPrepTest.scala

```diff
@@ -129,7 +127,7 @@ class KeystoneV2FilterPrepTest extends FunSpec with Matchers with MockitoSugar w
   describe("when the configuration is updated") {
     it("sets the current configuration on the filter asserting the defaults and initialized is true") {
       val filter = new KeystoneV2Filter(mock[ConfigurationService], mock[AkkaServiceClientFactory], mock[AtomFeedService], mockDatastoreService)
-      filter.isInitialized shouldNot be(right = true)
```

Comment:
Perhaps. Or perhaps IntelliJ suggested naming the boolean parameter, and I just went with it. I couldn't say any longer.

The real question is, why not simplify this assertion to:
`filter.isInitialized shouldBe false`
?